src/components/Header.js

```diff
@@ -0,0 +1,16 @@
+import './Header.css';
+
+function Header() {
+  return (
+    <div className="books_header">
+      <div className="nav">
+        <h3 className="dark_blue">Bookstore</h3>
+        <h4 className="gray_title">Books</h4>
+        <h4 className="fade_title">Categories</h4>
+      </div>
+      <div>profile</div>
+    </div>
+  );
+}
```

Comment:
- [ ] As stated in the project requirement **Your final design should match Zeplin's design (this includes all elements + the same styling)** So Kindly I recommend you to update your header for title 'Bookstore CMS' and  the 'Profile' Just like the given design **Kindly use an Person Icon just like the below Header design** .
- The Zeplin's design Header Look likes this : 

![header](https://github.com/MPDADDY/bookstore/assets/93598189/e8eb7887-9912-43bd-b504-d9ab2d5fd782)


- Your Header Design Look like this : 
![header](https://github.com/MPDADDY/bookstore/assets/93598189/455084a1-58fd-46a2-b8d4-6b0daa899dbb)